tests/behat/features/templates/about_us-subpages.feature

```diff
@@ -2,9 +2,10 @@
 @api
 @javascript
 
-Feature: test About Us menu subpages
+Feature: both unauthenticated and authenticated 
+users tests About Us menu subpages
```

Comment:
Goal is to have a longer description of what is being tested, in complete sentences.  But Hannah & I have been talking about this, trying to strike the balance between not too short & not too long.